kolibri/tasks/api.py

```diff
@@ -301,6 +302,9 @@ def canceltask(self, request):
         if 'task_id' not in request.data:
             raise serializers.ValidationError(
                 "The 'task_id' field is required.")
+        if type(request.data['task_id']) is not str:
```

Comment:
On my local server, I'm falling into this branch because the type is `unicode` instead of `str`

```
ipdb> request
<rest_framework.request.Request object at 0x10924b690>
ipdb> request.data
{u'task_id': u'7261de84d9bd4685a0955bc2d03331bb'}
ipdb> request.data['task_id']
u'7261de84d9bd4685a0955bc2d03331bb'
ipdb> type(request.data['task_id'])
<type 'unicode'>
ipdb>
```

Reply:
On python2.7? Maybe we don't have the right imports in this file.

Reply:
I added this line because of a completely unparseable error I was getting when and object got passed in instead of a task id.

Reply:
Have changed it to a py2/3 compatible test now, and added a test.